VATZ CI Error on `uninstall` & `enable`

### Checklist
- [x] Bug from Service (Vatz)
- [ ] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is.


### **How to Reproduce**
> Steps to reproduce the behavior:

1. Since Config register home_path, there's error on for following  CI command so tested with previous version of VATZ 
- `uninstall`
- `enable`

```
 git fetch origin pull/428/head:ISSUE-388-RES
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 22 (delta 13), reused 22 (delta 13), pack-reused 0
Unpacking objects: 100% (22/22), 9.71 KiB | 523.00 KiB/s, done.
From https://github.com/dsrvlabs/vatz
 * [new ref]         refs/pull/428/head -> ISSUE-388-RES
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 git checkout ISSUE-388-RES                  
A       .idea/.gitignore
Switched to branch 'ISSUE-388-RES'
 ./vatz plugin list                  
2023-05-21T21:27:48-05:00 INF List plugins module=plugin
2023-05-21T21:27:48-05:00 INF Create DB Instance module=db
2023-05-21T21:27:48-05:00 INF List module=plugin
2023-05-21T21:27:48-05:00 INF List module=db
+------+------------+--------------+------------+---------+
| NAME | IS ENABLED | INSTALL DATE | REPOSITORY | VERSION |
+------+------------+--------------+------------+---------+
+------+------------+--------------+------------+---------+
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   ISSUE-388-RES ±✚
 ./vatz plugin install github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor cpu-monitor1
2023-05-21T21:31:38-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor at /Users/dongyookang/.vatz module=plugin
2023-05-21T21:31:38-05:00 INF Create DB Instance module=db
2023-05-21T21:31:38-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor module=plugin
2023-05-21T21:31:41-05:00 INF AddPlugin module=db
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   ISSUE-388-RES ±✚
 ./vatz plugin enable cpu-monitor1 false                                                       
2023-05-21T21:31:50-05:00 INF enable installed plugin cpu-monitor1 at /Users/dongyookang/.vatz module=plugin
2023-05-21T21:31:50-05:00 INF Create DB Instance module=db
2023-05-21T21:31:50-05:00 INF UpdatePlugin module=db
2023-05-21T21:31:50-05:00 INF Plugin cpu-monitor1 has been updated: disabled. module=db
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   ISSUE-388-RES ±✚
 ./vatz plugin list                                                                                            
2023-05-21T21:32:06-05:00 INF List plugins module=plugin
2023-05-21T21:32:06-05:00 INF Create DB Instance module=db
2023-05-21T21:32:06-05:00 INF List module=plugin
2023-05-21T21:32:06-05:00 INF List module=db
+--------------+------------+---------------------+-------------------------------------------------------------+---------+
| NAME         | IS ENABLED | INSTALL DATE        | REPOSITORY                                                  | VERSION |
+--------------+------------+---------------------+-------------------------------------------------------------+---------+
| cpu-monitor1 | false      | 2023-05-21 21:31:41 | github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor | latest  |
+--------------+------------+---------------------+-------------------------------------------------------------+---------+

```

2. Switched to Current VATZ `main`

```
 clear
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 make build
go build -ldflags="-X 'github.com/dsrvlabs/vatz/utils.Version=main' -X 'github.com/dsrvlabs/vatz/utils.Commit=85df83a'" -v
github.com/dsrvlabs/vatz/manager/config
github.com/dsrvlabs/vatz/manager/api
github.com/dsrvlabs/vatz/manager/types
github.com/dsrvlabs/vatz/utils
github.com/dsrvlabs/vatz/manager/plugin
github.com/dsrvlabs/vatz/manager/dispatcher
github.com/dsrvlabs/vatz/monitoring/prometheus
github.com/dsrvlabs/vatz/manager/healthcheck
github.com/dsrvlabs/vatz/manager/executor
github.com/dsrvlabs/vatz/rpc
github.com/dsrvlabs/vatz/cmd
github.com/dsrvlabs/vatz
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 ./vatz init       
2023-05-21T21:36:47-05:00 INF init module=main
2023-05-21T21:36:47-05:00 INF create file default.yaml module=main
2023-05-21T21:36:47-05:00 INF home path ~/.vatz module=main
2023-05-21T21:36:47-05:00 INF Load Config default.yaml module=config
2023-05-21T21:36:47-05:00 INF Plugin dir /Users/dongyookang/.vatz module=main
2023-05-21T21:36:47-05:00 INF Create DB Instance module=db
2023-05-21T21:36:47-05:00 INF initDB /Users/dongyookang/.vatz/vatz.db module=db
2023-05-21T21:36:47-05:00 INF Remove old DB file module=db
2023-05-21T21:36:47-05:00 INF Create new DB file module=db
2023-05-21T21:36:47-05:00 INF Create DB Instance module=db
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 ./vatz plugin install github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor cpu-monitor1
2023-05-21T21:37:17-05:00 INF Load Config default.yaml module=config
2023-05-21T21:37:17-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor at /Users/dongyookang/.vatz module=plugin
2023-05-21T21:37:17-05:00 INF Create DB Instance module=db
2023-05-21T21:37:17-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor module=plugin
2023-05-21T21:37:19-05:00 INF AddPlugin module=db
 dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 ./vatz plugin uninstall cpu-monitor1                                                          
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1006a7b20]

goroutine 1 [running]:
github.com/dsrvlabs/vatz/cmd.glob..func5(0x100f32a60?, {0xc000119ed0, 0x1, 0x1?})
        /Users/dongyookang/ffplay/GolandProjects/dsrvlabs/vatz/cmd/plugin.go:117 +0x40
github.com/spf13/cobra.(*Command).execute(0x100f32a60, {0xc000119eb0, 0x1, 0x1})
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc000274780)
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(0x100edc088?)
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918 +0x19
main.main()
        /Users/dongyookang/ffplay/GolandProjects/dsrvlabs/vatz/main.go:19 +0x1e
 ✘ dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 
 ./vatz plugin enable cpu-monitor1 false
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1006a84a0]

goroutine 1 [running]:
github.com/dsrvlabs/vatz/cmd.glob..func10(0x100f331e0?, {0xc00007d660, 0x2, 0x2?})
        /Users/dongyookang/ffplay/GolandProjects/dsrvlabs/vatz/cmd/plugin.go:201 +0x40
github.com/spf13/cobra.(*Command).execute(0x100f331e0, {0xc00007d620, 0x2, 0x2})
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc000264780)
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(0x100edc088?)
        /Users/dongyookang/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918 +0x19
main.main()
        /Users/dongyookang/ffplay/GolandProjects/dsrvlabs/vatz/main.go:19 +0x1e
 ✘ dongyookang DK 😎   ~/ffplay/GolandProjects/dsrvlabs/vatz   main ±✚
 

``` 


### **Expected behavior**
> A clear and concise description of what you expected to happen.

working clear as plugin update enable ture/false and uninstall 


### **Screenshots**
> If applicable, add screenshots to help explain your problem.

--- a/cmd/plugin.go
+++ b/cmd/plugin.go
@@ -113,6 +113,10 @@ var (
 		Short:   "Uninstall plugin from plugin registry",
 		Args:    cobra.ExactArgs(1), // TODO: Can I check real git repo?
 		Example: "vatz plugin uninstall name",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			_, err := config.InitConfig(configFile)
+			return err
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			pluginDir, err := config.GetConfig().Vatz.AbsoluteHomePath()
 			if err != nil {
@@ -197,6 +201,10 @@ var (
 		Short:   "Enabled or Disable plugin",
 		Args:    cobra.ExactArgs(2), // TODO: Can I check real git repo?
 		Example: "vatz plugin enable <plugin_id> <true/false>",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			_, err := config.InitConfig(configFile)
+			return err
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			pluginDir, err := config.GetConfig().Vatz.AbsoluteHomePath()
 			if err != nil {

